packages/polar-betterauth/src/plugins/checkout.ts

```diff
@@ -100,7 +100,7 @@ export const checkout =
 							successUrl: checkoutOptions.successUrl
 								? new URL(
 										checkoutOptions.successUrl,
-										ctx.request?.url,
+                    ctx.request?.url ?? ctx.context.baseURL
```

Comment:
```suggestion
										ctx.request?.url ?? ctx.context.baseURL,
```

Formatting (we should fix the setup in this repo)